src/bridge/qos.go

```diff
@@ -5,13 +5,14 @@ import (
 	"fmt"
 	log "minilog"
 	"strconv"
+	"time"
 )
 
-// Used to calulate burst rate for the token bucket filter qdisc
+// Used for queue length in qdisc netem
 const (
-	KERNEL_TIMER_FREQ uint64 = 250
-	MIN_BURST_SIZE    uint64 = 2048
-	DEFAULT_LATENCY   string = "100ms"
+	MIN_NETEM_LIMIT     = 10
```

Comment:
Might be good to document how you got Min/Max/Default here. Also Go recommends MixedCaps rather than underscores [0] -- I just haven't gone through and renamed them everywhere else yet.

[0] https://golang.org/doc/effective_go.html#mixed-caps